db/schema.rb

```diff
@@ -26,6 +26,17 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "rents", force: :cascade do |t|
+    t.bigint "user_id"
+    t.bigint "book_id"
+    t.datetime "from"
```

Comment:
I think we forgot to update the schema.
You can do `rails db:drop db:create db:migrate`